lib/options/options.entry.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+
+// include the LICENSE file
+// $FlowIgnore
+import 'file-loader?name=LICENSE!../../LICENSE';
```

Comment:
This creates a file in the output dir so we only need it in `foreground.entry.js`